### PR TITLE
Added compatibility for using APS1

### PR DESCRIPTION
**Pull request by Himorask**
Modified set_measure and set_control to be compatible with the hardware configuration of APS1.

### CODE REVIEW

File: QGL/ChannelLibraries.py [hunk 1]
```diff
@@ -389,12 +389,16 @@ def new_source(self, label, model, address, power=-30.0, frequency=5.0e9, refere
...
 
-    def set_control(self, qubit_or_edge, transmitter, generator=None):
+    def set_control(self, qubit_or_edge, transmitter, generator=None, aps1_chan=None):
         quads   = [c for c in transmitter.channels if isinstance(c, Channels.PhysicalQuadratureChannel)]
         markers = [c for c in transmitter.channels if isinstance(c, Channels.PhysicalMarkerChannel)]
 
         if isinstance(transmitter, Channels.Transmitter) and len(quads) > 1:
-            raise ValueError("In set_control the Transmitter must have a single quadrature channel or a specific channel must be passed instead")
+            #raise ValueError("In set_control the Transmitter must have a single quadrature channel or a specific channel must be passed instead")
+            print("Warning: In set_control the Transmitter must have a single quadrature channel if using APS2. Assuming APS1.")
```
Comment (dieris) [inline QGL/ChannelLibraries.py]: I'd use `warn` from `warnings` module

File: QGL/ChannelLibraries.py [hunk 2]
```diff
@@ -389,12 +389,16 @@ def new_source(self, label, model, address, power=-30.0, frequency=5.0e9, refere
...
+            if aps1_chan != '12' and aps1_chan != '34':
```
Comment (dieris) [inline QGL/ChannelLibraries.py]: How about redefining channel conventions like for APS2:
https://github.com/BBN-Q/QGL/commit/5fd7a903caec83ef615c2ca14f3e31573af7277c

Comment (dieris) [reply QGL/ChannelLibraries.py]: then it would be something like `if int(aps1_chan) > 2`:

**Comment by dieris** [comment]
Great, I'm happy to see that changes were faster than my comments :)

File: QGL/ChannelLibraries.py (hunk 1 above)
Comment (Himorask) [reply QGL/ChannelLibraries.py]: After talking to Graham, we kept the error message. It turns out you can access the main transmitter back from the single channel, so we do not need to pass the whole APS1 transmitter.

**Pull request merged**